### ISSUE TITLE
Noise remover in audio edition [sf#1953796]

### ISSUE THREAD

**Issue by Windfisch** [muse-sequencer/muse#271]
Submitted by cargobidibulle on 2008-04-29 03:14:26

I wish there were a noise remover function in audio edition, such as that of cooledit (windows) or gnome wave cleaner (there is also one in audacity but it does not work very well). Such noise remover works on recording a noise sample, and then removing its trace from the audio.

For me, this is the second most important function of audio edition, after controls of volume.

**Comment by Reaper10** [comment]
is this still not going to happen?

**Comment by spamatica** [comment]
Hello Reaper10,

It's very nice with your enthusiasm with adding more things into MusE, we
would very much like to add a great many of the things you suggest.
But, we are a very small team who work on MusE (and personally I fear I
have problems finding time for MusE at all, for the moment) so we can't add
that many things on our own.

We tend to prioritise things that will improve MusE in a way that would be
much harder to achieve by using other software. Like if there already is a
plugin for noise-removal that can be run from inside MusE, then we would
likely not spend time on adding an internal solution, unless the plugin was
garbage.
Now, I'm not sure there is a plugin, I know there is external tools that
can do this so it is possible but it would be easier inside MusE... so
there might be reason to add this but I fear we do not prioritize it.

What will always work if some new developer show up that is interested in
doing the implementation, we happily accept patches and other additions.
Secondly, we have a donate page for helping us money-wise, maybe we should
add a bounty-style payment possibility, where people can come together and
basically pay for having some specific feature to be added, a paid feature
would always be added. ...though I'm not sure MusE has big enough user-base
to make that a workable scenario.

I CC:d the mailinglist as atleast the last paragraph is of general interest.

Regards,
Robert

2017-01-28 22:36 GMT+01:00 Reaper10 <notifications@github.com>:

**Comment by Reaper10** [comment]
I'm sorry for bothering you I know not much help I can't code.

**Comment by spamatica** [comment]
You are not bothering me, it is nice with enthusiastic users :)

I was mostly trying

2017-01-29 17:44 GMT+01:00 Reaper10 <notifications@github.com>:

**Comment by Reaper10** [comment]
well I can't use muse I use windows

**Comment by terminator356** [comment]
On Sunday, January 29, 2017 9:24:04 AM EST Reaper10 wrote:
> well I can't use muse I use windows

For the technical folks interested...

I have MusE about 70% compiled on Windows 7.

However, there are some problems.

Chiefly, the build environment:
At first I tried a pure Windows build, but it may be 
 a while before we can get to that point.
(Hence the flurry of recent MusE activity attempting to
 remove some of the POSIX stuff from MusE.)

The way at the moment is building under MSYS-2.

However, I had great difficulty understanding what's up with that:
MSYS-2 has MINGW environments complete with Qt - but they 
 do not support POSIX which MusE currently needs.
Conversely, in the MSYS-2 environment POSIX is supported
 but there is no Qt and I could not figure out what to do.
The instructions indicate you can build Qt but apparently
 the build goes into the MINGW environments, not the MSYS-2
 environment.

So I'm a bit stuck on how to use POSIX MSYS-2 right now.
I am therefore concentrating on removing or replacing
 the POSIX stuff in MusE, so that a pure Windows build
 might be possible in the future.

Tim.

**Comment by Reaper10** [comment]
what about windoes 10

**Comment by terminator356** [comment]
On Sunday, January 29, 2017 10:41:09 AM EST Reaper10 wrote:
> what about windoes 10

What the heck is windoes ? Don't know that one. Spellcheck.

Did I not just say I'm working on it?
I appreciate your enthusiasm but please, get serious.
On this issue list we can deal with actual bugs, 
 and reasonable suggestions and wishes. Not this.

If you run only Windows why are you submitting 'drive-by' suggestions 
 to a Linux project? Have you ever actually run Linux, or MusE?

Please, know what you are dealing with before making suggestions.

And FYI, yes Windows 10 supposedly has Linux support.
But I only have Windows 7 right now.

Tim.
